audience-analytics/python-app.rst

```diff
@@ -0,0 +1,48 @@
+.. _apps:
+
+Client apps
++++++++++++
+.. after-title
+
+Our client apps are lightweight, open source scripts or executable, that you will need to install, configure and run on a your computer.
+
+They perform two basic tasks:
+
+1. retrieve a valid video stream. By default, one of the connected USB camera will be chosen, but you can easily modify the client app to open a different camera and even open a video file.
+
+2. package and sent the video stream over http to our computation servers. This part can also be optimized for your needs (image resolution, frame rate, etc...).
```

Comment:
I think 'send' is better than 'sent' and use 'https' instead of 'http'

Reply:
Fixed